runtime/runtime/src/lib.rs

```diff
@@ -260,6 +260,7 @@ impl Runtime {
                         receipt_ids: vec![receipt.receipt_id],
                         gas_burnt: verification_result.gas_burnt,
                         tokens_burnt: verification_result.burnt_amount,
+                        receiver_id: transaction.receiver_id.clone(),
```

Comment:
This is not right. This should be the account on which the execution happened. In this case the `signer_id`